### ISSUE TITLE
Emojis added to the server in every bot restart

### ISSUE THREAD

**Issue by abhinavtk97** [denverquane/automuteus#107]
In each bot restart, all the emojis are added again, filling up the whole emoji slots
Edit: the bot was added to 2 servers and one of them had all the emojis while the other had only some (the first set of emojis that was added while in development without dead emojis). I had not specified the EMOJI_GUILD_ID before and it was defaulting to the guild without full emojis. In the second server without the full emojis, the administrator permission was removed a while ago and so, the bot is not able to add emojis. So,  every time the bot is restarted, the emojis are getting added to both servers and gets added in the server with admin permission and it fails in the server without permissions.

By setting EMOJI_GUILD_ID specifically, i could prevent this from happening, but the emojis should only be added to the server where emojis are not present and not to all the servers where the bot is a member

**Comment by ShawnHardwick** [comment]
I can confirm a similar and maybe more simplified scenario.

I invited version 2.2.4 of the bot to my test server where it imported the emojis. After testing that it worked, I invited the bot to my public server. When creating a new Among Us session with .au new on the public server, it would reference the test server's emojis. 

This seemed to be fine until I restarted my docker container running the bot. The emoji's were then added to the public server, but on the test server there are now duplicates of each emoji. I'm concerned over restarting it again because it might keep adding more emoji's.
